composer.json

```diff
@@ -11,7 +11,7 @@
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "league/oauth1-client": "^1.9",
-        "socialiteproviders/manager": "~4.0"
+        "socialiteproviders/manager": "^4.2"
```

Comment:
please bump this for the provider not for the root composer json :)

Reply:
Oops! Good catch. Sorry for the error.

Reply:
Looks like its still in the root json

Reply:
I believe this has been resolved now?